shinysdr/plugins/rtty/__init__.py

```diff
@@ -55,59 +56,105 @@
 
 
 class RTTYDemodulator(gr.hier_block2, ExportedState):
+    '''Demodulate typical amateur RTTY.
+
+    Assumptions:
+
+        - 45.45 baud
+        - 170 Hz spacing
+        - Mark tone high
+
+    TODO: make these assumptions parameters.
+    '''
+
     implements(IDemodulator)
     
-    __filter_low = 1500
-    __filter_high = 2500
-    __transition = 100
+    _spacing = 170
+
+    _low_cutoff = _spacing * -2
```

Comment:
A comment explaining how these values arise would be good. For that matter, "input should be centered on the mark tone" ought to be documented somehow — as we discussed, best by a hairline, but since that's not currently convenient, at least in the class docstring here.

…Actually, you could add a second hairline kludgily like so:

    class SecondHairline(ExportedState):
        implements(IHasFrequency)
        @exported_value(type=float, changes='explicit')
        def get_freq(self):
            return self.__freq
    
        def set_freq(self, freq):
            self.__freq = freq
            self.state_changed('freq')

and then have the demodulator object contain one of these (`@exported_value(type=ReferenceT())`). But maybe leave that for me to do.